matrix-sdk-android/src/main/res/values/strings_riotX.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!-- Strings not defined in Riot -->
+    <string name="in_reply_to">In reply to</string>
```

Comment:
All those strings already exists, please remove this file

Reply:
The strings have been moved from app to sdk. Will delete the ones in app

Reply:
thx